scripts/handleFailure.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#/******************************************************************************
+# * Licensed Materials - Property of IBM
+# * IBM Cloud Container Service, 5737-D43
+# * (C) Copyright IBM Corp. 2017, 2018 All Rights Reserved.
+# * US Government Users Restricted Rights - Use, duplication or
+# * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# *****************************************************************************/
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_GO_VERSION" == "tip" ]; then
+	curl -s -k -X GET -H "Content-Type: application/json" -H "Accept: application/vnd.travis-ci.2+json"  -H "Authorization: token $TRAVIS_TOKEN"  https://travis.ibm.com/alchemy-containers/armada-storage-file-plugin/builds/$TRAVIS_BUILD_ID | jq '.jobs[0].state' | sed 's/"//g'> state.out
```

Comment:
same here, please check this looks it will not work for public github.com

Reply:
Need to work on it.

I am going to work on it.